NumWords.java

```diff
@@ -2,5 +2,11 @@
 public class NumWords {
 	public static void main(String args[]) {
 	    // Replace this comment with your code
+		int n = Integer.parseInt(args[0]);
+		int h = n/100;
+		int t = n%100/10;
```

Comment:
Names of variables should be indicative. 
Lack of spacing between mathematical operations.